### DISCUSSION TITLE
[new feature request] openssl AES GMAC support

### DISCUSSION THREAD

**Discussion by pankajbansal3073** [strongswan/strongswan#2398] [Ideas]
Hi,

I am trying to use strongswan with openssl plugin to test AES GMAC algorithm. As i can see that openssl (or for that matter any plugin) is providing the support for this algorithm in strongswan. can you please add support for this algo in openssl plugin ?

**Comment by strongX509** [comment]
The strongSwan project is about **strong** security. This is why we don't put much priority on crypto algorithms with _null encryption_.
